Organization/Organization/CreateEmployeeViewController.m

```diff
@@ -0,0 +1,60 @@
+//
+//  CreateEmployeeViewController.m
+//  Organization
+//
+//  Created by New user on 02.08.17.
+//  Copyright © 2017 New user. All rights reserved.
+//
+
+#import "CreateEmployeeViewController.h"
+#import "MainViewController.h"
+@interface CreateEmployeeViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *firstName;
```

Comment:
In `firstName` property I expect to see a string, but not a textfield. Rename to firstNameTextField.